### ISSUE TITLE
Hi-Z input of SDRplay2 (soapy) not usable

### ISSUE THREAD

**Issue by helioshk** [csete/gqrx#697]
Antenna 1 and antenna 2 inputs work perfectly. When selecting Hi-Z input, the data stream stops.
I don't know if this is a bug in gqrx or soapy. Let me know if and how I can provide more information.

**Comment by csete** [comment]
Indeed, three is no SDRplay or any other device-specific code in gqrx. It just reads and processes parameters read from the backend. So the issue could be in [soapysdrplay](https://github.com/pothosware/SoapySDRPlay). You can perhaps test with cubicsdr or other SDR application that uses soapysdr.
